source/opt/trim_capabilities_pass.cpp

```diff
@@ -151,10 +151,33 @@ Handler_OpTypePointer_StorageInputOutput16(const Instruction* instruction) {
              : std::nullopt;
 }
 
+static std::optional<spv::Capability>
+Handler_OpTypePointer_StoragePushConstant16(const Instruction* instruction) {
+  assert(instruction->opcode() == spv::Op::OpTypePointer &&
+         "This handler only support OpTypePointer opcodes.");
+
+  // This capability is only required if the variable as an Input/Output storage
```

Comment:
// This capability is only required if the variable has a PushConstant storage class

Reply:
Ah yes, thanks!